packages/main-ui/src/pages/my_profile/page.rs

```diff
@@ -132,27 +132,28 @@ pub fn MissionHistoryComponent(lang: Language, histories: Vec<MissionHistory>) -
     let tr: MissionHistoryComponentTranslate = translate(&lang);
     let ctrl = MissionHistoryController::new()?;
     let responsive: ResponsiveService = use_context();
+
     rsx! {
         div {
-            class: "flex flex-col justify-start items-start bg-transparent border border-[#e0e0e0] rounded-[12px]",
-            style: if responsive.width() > 1200.0 { "width: 100%;" } else { "width: 110px; height: 100%" },
-            div { class: if responsive.width() > 1200.0 { "flex flex-row w-full h-[45px] justify-start items-start font-semibold text-[16px] text-[#636363] border-b border-b-[#e0e0e0]" } else { "flex flex-col h-full justify-start items-start font-semibold text-[14px] text-[#636363] border-b border-b-[#e0e0e0]" },
-                div { class: "flex flex-1 justify-start items-center px-[20px] py-[10px]",
+            class: "flex flex-col justify-start items-start bg-transparent border border-[#e0e0e0] rounded-[12px] overflow-x-auto",
+            style: if responsive.width() > 1200.0 { "width: 100%;" } else { "width: 100%; min-width: 300px;" },
+            div { class: "flex flex-row w-full h-[45px] justify-start items-start font-semibold text-[16px] text-[#636363] border-b border-b-[#e0e0e0]",
```

Comment:
```suggestion
            class: "flex flex-col justify-start items-start bg-transparent border border-[#e0e0e0] rounded-[12px] overflow-x-auto",
            div {
                  class: "w-full min-w-[300px]" ,
                      div { class: "flex flex-row w-full h-[45px] justify-start items-start font-semibold text-[16px] text-[#636363] border-b border-b-[#e0e0e0]",
                      .....
               }
```

At this, you may not need to use style. it can complete by tailwindcss. 
And I think `overflow-x-scoll` may wrap a `div` wrapping table.

@Nathy-bajo check it out and attach sccrennshot

Reply:
@hackartists Yes, after reviewing it again, using TailwindCSS for responsiveness and avoiding inline styles makes the code more consistent and maintainable.

Thank you for the suggestion!